deployments/moonriver.diamond.json

```diff
@@ -33,6 +33,10 @@
         "Name": "PeripheryRegistryFacet",
         "Version": "1.0.0"
       },
+      "0x50b4cB2C152322760a8fA59105A46745CE2B7Eac": {
```

Comment:
wondering why we dont have the info for this contract in the log....shouldnt be, right?